extensions/amp-script/validator-amp-script.protoascii

```diff
@@ -198,11 +198,9 @@ tags: {  # <amp-script>
       also_requires_attr: "script"
     }
   }
-  attrs: {
-    name: "nodom"
-    value: ""
```

Comment:
I think you actually want the empty value here as well as for sandboxed? Your code appears to treat this a boolean, so allowing values seems incorrect. Is that accurate?

Reply:
Good point!